experiments/compositions/composition/internal/controller/composition_controller.go

```diff
@@ -47,9 +47,15 @@ type CompositionReconciler struct {
 //+kubebuilder:rbac:groups=composition.google.com,resources=compositions,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=composition.google.com,resources=compositions/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=composition.google.com,resources=compositions/finalizers,verbs=update
-//+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list
+//+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list;watch
 //+kubebuilder:rbac:groups="",resources=events,verbs=create;patch
+//+kubebuilder:rbac:groups=alice.alice,resources=*,verbs=get;list;watch;update;patch
```

Comment:
Not a blocker, but we probably want to include the alice permissions separately (as part of CRD creation?)

Reply:
Agreed. Trying to figure this part out. Should we have a separate CRD that generates Roles or ask the user to edit a role.

Reply:
Ah i see.. part of alice crd onboarding is appropriate place.